daml/Tests/Distribution/Syndication/Hedging.daml

```diff
@@ -25,11 +25,11 @@ test = do
   deposit operator bondRegistrar custodian1 100_000_000.0 swap1.id
   deposit operator custodian1    investor1  100_000_000.0 (swap1.id with signatories = singleton custodian1)
 
-  submitMulti [operator, bondRegistrar] [] do createCmd Lifecycle.Today with operator; provider = bondRegistrar; date = date 2021 Jan 15
-  submitMulti [operator, bondRegistrar] [] do createCmd Lifecycle.Observation with operator; provider = bondRegistrar; label = libor6m.label; date = date 2022 Jun 21; value = 0.003
-  submitMulti [operator, bondRegistrar] [] do createCmd Lifecycle.Observation with operator; provider = bondRegistrar; label = libor6m.label; date = date 2022 Dec 20; value = 0.002
-  submitMulti [operator, bondRegistrar] [] do createCmd Lifecycle.Observation with operator; provider = bondRegistrar; label = libor6m.label; date = date 2023 Jun 20; value = 0.004
-  submitMulti [operator, bondRegistrar] [] do createCmd Lifecycle.Observation with operator; provider = bondRegistrar; label = libor6m.label; date = date 2023 Dec 20; value = 0.001
+  submitMulti [operator, payingAgent] [] do createCmd Lifecycle.Today with operator; provider = payingAgent; date = date 2021 Jan 15
```

Comment:
Paying agent is likely not the party feeding observations into the platform. But we can change that in a subsequent PR and have the paying agent just be an observer. All data feeding should probably be abstracted behind a DataService anyway. I'll look into this.

Reply:
@GeorgSchneider noted and thanks for pointing this out, I will bear in mind for the potential change in later stage.
So just curious who should be the one creating observation? bondRegistrar still?

Reply:
We haven't discussed yet - I'd assume it's probably the operator driving this